app/jobs/migrate_notifications_job.rb

```diff
@@ -0,0 +1,59 @@
+class MigrateNotificationsJob < ApplicationJob
```

Comment:
Looks good! Some tips I have for jobs (long-running jobs in general):
1. Want to be able to manually set offsets manually
    - in case a certain set of notifications break the job (e.g. the first 10 break, then we can skip those)
    - it's just generally good to have that kind of control without having to create a PR to change job params
2. Exception Handling for Migration Job (Safe Failures)
    - in most cases you don't want a failure to stop processing entirely - you want to log the failure but keep going
3. Logging 
     - log the happy path, but also log any skipped, and obviously log the errors. Ideally the stats would be summarized at the end of the job, so we can be confident we know what happened to each of the 1.5 million notifications (e.g. 1.49M migrated, 200 skipped, 5 errored) e.g.